src/libraries/System.Security.Permissions/tests/PermissionTests.cs

```diff
@@ -210,13 +210,6 @@ public static void PrincipalPermissionCallMethods()
             pp.ToXml();
         }
 
-        [Fact]
-        public static void PrincipalPermissionAttributeCallMethods()
-        {
-            PrincipalPermissionAttribute ppa = new PrincipalPermissionAttribute(new Permissions.SecurityAction());
-            IPermission ip = ppa.CreatePermission();
-        }
```

Comment:
Since existing binaries will continue to work, seems valuable to continue to validate the behavior.  Do we not have any way to write the test given the error obsoletion?

Reply:
Could put a test binary in the test data repo (where we put compression test data)?

Reply:
These methods quite literally no-op. If you attempt to use them for real you'll get a `NullReferenceException`. IMO this test (and the tests for `FileIOPermissionAttribute`, etc.) isn't testing anything useful.

Reply:
If there's really a desire to keep this test, we could always `Activator.CreateInstance` it, since that won't produce the build error. But again, [since the implementations are no-ops](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Security.Permissions/src/System/Security/Permissions/PrincipalPermissionAttribute.cs), I'm not sure what utility such a test has.

I'll defer to your collective judgment if you feel that such a test is useful. :)